docs/Options/GenerateOptions.md

```diff
@@ -1,7 +1,9 @@
 # Generate Options
 Option switches are case-sensitive, arguments are case-insensitive
 
-* `--violate` 
+* `--overwrite`
+    * Overwrite existing output files. Options are: `true` or `false` (default)
```

Comment:
Possible discussion point: We have removed the true/false parts from the other boolean command line switches as they don't need to be set on the commandline so thought it was clearer without the "Options are: `true` or `false` (default)"?

Reply:
yes, I copied this while the true/false options were there and didn't think they applied. 
I'll remove them